files/image_config/hostcfgd/hostcfgd

```diff
@@ -23,9 +23,9 @@ TACPLUS_SERVER_TIMEOUT_DEFAULT = "5"
 TACPLUS_SERVER_AUTH_TYPE_DEFAULT = "pap"
 
 
-def is_valid_hostname(name):
+def is_valid_hostname(hostname):
     if hostname[-1] == ".":
-        hostname = hostname[:-1] # strip exactly one dot from the right, if present
+        return False
     if len(hostname) > 253:
```

Comment:
You can now combine these into one statement:
```
if hostname[-1] == "." or len(hostname) > 253:
    return False
```

Reply:
OK